test/cliplugin/localkms/implementation.go

```diff
@@ -125,6 +128,39 @@ func (i LocalSignerVerifier) SignMessage(message io.Reader, opts ...signature.Si
 	return signature, nil
 }
 
+// VerifySignature verifies the signature.
+func (i LocalSignerVerifier) VerifySignature(signature io.Reader, message io.Reader, opts ...signature.VerifyOption) error {
+	privateKey, err := loadRSAPrivateKey(i.keyResourceID)
+	if err != nil {
+		return err
+	}
+	publicKey := privateKey.PublicKey
+
+	var digest []byte
+	var signerOpts crypto.SignerOpts = i.hashFunc
+	for _, opt := range opts {
+		opt.ApplyDigest(&digest)
+		opt.ApplyCryptoSignerOpts(&signerOpts)
+	}
+
+	if len(digest) == 0 {
```

Comment:
it's a bit confusing that the user can provide message or digest here. I would think it's one or the other?

Reply:
more specifically, I don't think the user should be able to specify both (because it could be error prone?)

Reply:
I think I agree, but since the interface allows both, we have to respect that for now.

Reply:
alright, can we leave an issue open about this? I feel like if we're gonna muck with the API, now would be the time.

Reply:
created https://github.com/sigstore/sigstore/issues/1949